lib/crypto/src/crypto.erl

```diff
@@ -642,87 +696,96 @@ nif_stub_error(Line) ->
 -ifndef(CRYPTO_VSN).
 -define(CRYPTO_VSN, "??").
 -endif.
+
 version() -> ?CRYPTO_VSN.
 
--spec start() -> ok | {error, Reason::term()}.
+-spec start() -> ok | {error, Reason :: term()}.
 start() ->
     case application:start(crypto) of
         ok ->
-            _ = supports(curves), % Build curves cache if needed
+            % Build curves cache if needed
+            _ = supports(curves),
             ok;
         Error ->
             Error
     end.
 
--spec stop() -> ok | {error, Reason::term()}.
+-spec stop() -> ok | {error, Reason :: term()}.
 stop() ->
     application:stop(crypto).
 
--spec supports() -> [Support]
-                        when Support :: {hashs,   Hashs}
-                                      | {ciphers, Ciphers}
-                                      | {public_keys, PKs}
-                                      | {macs,    Macs}
-                                      | {curves,  Curves}
-                                      | {rsa_opts, RSAopts},
-                             Hashs :: [sha1() | sha2() | sha3() | blake2() | ripemd160 | compatibility_only_hash()],
-                             Ciphers :: [cipher()],
-                             PKs :: [rsa | dss | ecdsa | dh | ecdh | eddh | ec_gf2m],
-                             Macs :: [hmac | cmac | poly1305],
-                             Curves :: [ec_named_curve() | edwards_curve_dh() | edwards_curve_ed()],
-                             RSAopts :: [rsa_sign_verify_opt() | rsa_opt()] .
+-spec supports() -> [Support] when
+    Support ::
+        {hashs, Hashs} |
+        {ciphers, Ciphers} |
+        {public_keys, PKs} |
+        {macs, Macs} |
+        {curves, Curves} |
+        {rsa_opts, RSAopts},
```

Comment:
The original shows that there alternatives just by the aligned vertical bars in front, while you have to read all to understand that in the new formatting.

Reply:
Thanks again for pointing this out.  Just for reference here is the issue where we can collect the feedback about vertical bars or pipes https://github.com/WhatsApp/erlfmt/issues/167